README.md

```diff
@@ -30,71 +30,88 @@ The Getting Started instructions can be found [here](GettingStarted.md).
+| Scaling Bottlenecks | Batchers, Execution/Routing Engines, and/or other middlemen | Middlemen can take advantage of their position between users and the protocol. Even if MEV is mitigated, more users --> more execution demand --> centralization of middlemen |
 
-Each of these downsides have their own corresponding "solution": yield farming, governance tokens, and batchers, respectively. However, these "solutions" have downsides of their own:
- 
-  1. The long term sustainability of yield farming is questionable.
-  2. Fairly distributing governance tokens is difficult.
-  3. Batchers are effectively middle-men that can take advantage of their position between users and the protocol.
+The key takeaway here (especially from the last point) is even 
 
-Even more concerning is that, even if governance tokens are fairly distributed, this doesn't change the fact that a liquidity pool can only be delegated to one stake pool. Fractionalizing liquidity pools helps but it still results in a more centralized PoS blockchain than if each user was able to delegate their assets independently.
+Of course, this is not an exhaustive list, but here is the key takeaway: **even if users control their own keys, for a DEX to achieve true scale the users must also control their own *addresses*.**
+
+
+if some workarounds *can* be effective, the underlying design *principles* are inadequate. Much like Bittorrent, and the CSL-CCL stack, **the best p2p protocols are ones that scale in proportion to the number of participants**. DEXes are no different. With this in mind, we can explore alternative design principles:
 
 ### Programmable Swaps
-Programmable Swaps are a more promising design than liquidity pools and were first proposed by Axo (formerly known as Maladex). However, in Axo's [whitepaper](https://www.axo.trade/whitepaper.pdf), there is no mention of giving users full delegation control of their own assets. When they were asked on Discord about it, they said it was possible but delegation control would not be included in the first version.
+Programmable Swaps are a better design choice than liquidity pools. They were first proposed by Axo (formerly known as Maladex). However, in Axo's original [whitepaper](https://www.axo.trade/whitepaper.pdf), there is no mention of giving users *full* delegation control of their own assets. 
 
-### The Cardano-Swaps DEX
-Cardano-Swaps took inspiration from Axo's programmable swaps design but added delegation control as a foundational feature. `cardano-swaps` is the name of the CLI program included to help use the DEX.
+### The cardano-swaps Protocol
+cardano-swaps took inspiration from Axo's programmable swaps design but added delegation control as a foundational feature. `cardano-swaps` is the name of the CLI program that enables use of the DEX protocol.
```

Comment:
Should "cardano-swaps" be capitalized here? The README header has it capitalized. I originally used Cardano-Swaps as the name of the DEX and `cardano-swaps` as the name of the helper CLI.